src/SFA.DAS.PR.Jobs/Functions/Notifications/SendNotificationsFunction.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.PAS.Account.Api.Types;
+using SFA.DAS.PR.Data;
+using SFA.DAS.PR.Data.Common;
+using SFA.DAS.PR.Data.Entities;
+using SFA.DAS.PR.Data.Repositories;
+using SFA.DAS.PR.Jobs.Configuration;
+using SFA.DAS.PR.Jobs.Infrastructure;
+using SFA.DAS.PR.Jobs.Services;
+
+namespace SFA.DAS.PR.Jobs.Functions.Notifications;
+
+public class SendNotificationsFunction
+{
+    private readonly ILogger<SendNotificationsFunction> _logger;
+    private readonly NotificationsConfiguration _notificationsConfiguration;
+    private readonly IFunctionEndpoint _functionEndpoint;
+    private readonly IProviderRelationshipsDataContext _providerRelationshipsDataContext;
+    private readonly INotificationRepository _notificationRepository;
+    private readonly ITokenService _tokenService;
+    private readonly IPasAccountApiClient _pasAccountApiClient;
+
+    public SendNotificationsFunction(
+        ILogger<SendNotificationsFunction> logger,
+        IConfiguration configuration,
+        IFunctionEndpoint functionEndpoint,
+        IProviderRelationshipsDataContext providerRelationshipsDataContext,
+        INotificationRepository notificationRepository,
+        ITokenService tokenService,
+        IPasAccountApiClient pasAccountApiClient
+    )
+    {
+        _logger = logger;
+        _functionEndpoint = functionEndpoint;
+        _providerRelationshipsDataContext = providerRelationshipsDataContext;
+        _notificationRepository = notificationRepository;
+        _tokenService = tokenService;
+        _pasAccountApiClient = pasAccountApiClient;
+
+        _notificationsConfiguration = new NotificationsConfiguration();
+        configuration.GetSection("ApplicationConfiguration:Notifications").Bind(_notificationsConfiguration);
```

Comment:
Since the function is running every 5 minutes, I would use IOptions pattern to register ApplicationConfiguration and have dependency on IOptions here so we are not reading the configuration object and deserialising as many times the function runs.